doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1005,7 +1005,162 @@ Response code:
             "temperature"
         ]
 
-## Standard OMA-NGSI operations [/v2/queryContext]
+## Context Subscriptions [/v2/entities/{entityId}/subscriptions]
+
+This section is still under discussion.
+
+A subscription is represented by a JSON object with the following fields:
+
+* 'id': Subscription unique identifier
```

Comment:
It should be said that `id` cannot be set by the client, i.e. it is forbidden in the POST request to create subscriptions and will be provided in the GET request to get the list.

(BTW is better to use code style for things that appear literarly in the JSON, with ` instead of ')